ludwig/utils/gbm_utils.py

```diff
@@ -0,0 +1,195 @@
+from typing import Callable, Dict, Iterable, Tuple
+
+import lightgbm as lgb
+import numpy as np
+import torch
+from lightgbm.callback import CallbackEnv
+from numpy import typing as npt
+
+from ludwig.constants import NUMBER
+from ludwig.features.base_feature import BaseFeatureMixin, OutputFeature
+from ludwig.features.category_feature import CategoryOutputFeature
+from ludwig.features.feature_utils import LudwigFeatureDict
+from ludwig.models.base import BaseModel
+
+
+def iter_feature_metrics(features: LudwigFeatureDict) -> Iterable[Tuple[str, str]]:
```

Comment:
I believe this is unused and can be deleted.

Reply:
Nice catch - removed